code/development-env/config/local/image_config.json

```diff
@@ -0,0 +1,97 @@
+{
+  "images": {
+    "RSHINY": {
+      "displayName": "Shiny",
+      "type": "site",
+      "userSelectableVersion": true,
+      "versions": [
+        {
+          "default": true,
+          "displayName": "4.0.1",
+          "image": "nerc/rshiny:4.0.1"
+        },
+        {
+          "displayName": "3.5.3",
+          "image": "nerc/rshiny:3.5.3"
+        }
+      ]
+    },
+    "RSTUDIO": {
+      "displayName": "RStudio",
+      "type": "notebook",
+      "userSelectableVersion": true,
+      "versions": [
+        {
+          "default": true,
+          "displayName": "4.0.1",
+          "image": "nerc/rstudio:4.0.1",
+          "connectVersion": "nerc/zeppelin-connect:1.1.1"
+        },
+        {
+          "default": false,
+          "displayName": "3.5.3",
+          "image": "nerc/rstudio:3.5.3",
+          "connectVersion": "nerc/zeppelin-connect:1.1.1"
+        }
+      ]
+    },
+    "JUPYTERLAB": {
+      "displayName": "JupyterLab",
+      "type": "notebook",
+      "versions": [
+        {
+          "displayName": "f200ab964cea-dask-2.30.0",
+          "image": "nerc/jupyterlab:f200ab964cea-dask-2.30.0"
+        }
+      ]
+    },
+    "JUPYTER": {
+      "displayName": "Jupyter",
+      "type": "notebook",
+      "versions": [
+        {
+          "displayName": "f200ab964cea-dask-2.30.0",
+          "image": "nerc/jupyterlab:f200ab964cea-dask-2.30.0"
+        }
+      ]
+    },
+    "ZEPPELIN": {
+      "displayName": "Zeppelin",
+      "type": "notebook",
+      "versions": [
+        {
+          "displayName": "0.7.2.7",
+          "image": "nerc/zeppelin:0.7.2.7",
+          "connectVersion": "nerc/zeppelin-connect:1.1.1"
+        }
+      ]
+    },
+    "NBVIEWER": {
+      "displayName": "NBViewer",
+      "type": "site",
+      "versions": [
+        {
+          "displayName": "latest",
+          "image": "jupyter/nbviewer:latest"
+        }
+      ]
+    },
+    "SPARK": {
+      "image": "nerc/spark-k8s:0.1.0",
+      "masterAddress": "spark://spark-master:7077",
+      "sharedRLibs": "/data/packages/R/%p/%v"
+    },
+    "DASK": {
+      "image": "daskdev/dask:2.30.0"
```

Comment:
Is it worth keeping the `image` for Dask and Spark in a `versions` array to keep it consistent with the other images?